src/components/Pages/Books.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../Styles/Books.css';
+import PropTypes from 'prop-types';
+
+const Books = ({ book }) => {
+  Books.propTypes = {
+    book: PropTypes.shape({
+      category: PropTypes.string.isRequired,
+      title: PropTypes.string.isRequired,
+      author: PropTypes.string.isRequired,
+    }),
+  };
+  Books.defaultProps = {
+    book: null,
+  };
+  return (
+    <div>
+      <div className="book-container">
+        <div className="part-one">
+          <div>
+            <p className="catagory">{book.category}</p>
+            <h2 className="title">{book.title}</h2>
+            <p className="author">{book.author}</p>
+          </div>
+
+          <div className="button-nav">
+            <ul>
+              <button className="noborder" type="button">Comment</button>
+              <button className="noborder" type="button">Remove</button>
+              <button className="noborder" type="button" style={{ border: 'none' }}>Edit</button>
+            </ul>
```

Comment:
- [ ] Kindly make sure to follow the best practices. It is not allowed to `ul` to have any other children other than `li` but you have even added the `button` tag as the children of `ul`. Kindly make sure to wrap your button into the `li` tag. :wink: